src/components/CareerEvent/CareerEvent.js

```diff
@@ -1,5 +1,30 @@
 import React from "react";
 
-export default function CareerEvent() {
-  return <div className="careerEvent"></div>;
-}
+//STYLE dependencies
+import { TableRow, TableCell } from "@material-ui/core";
+
+const CareerEvent = ({
+  careerEventID,
+  name,
+  date,
+  description,
+  location,
+  link
+}) => (
+  <TableRow>
+    <TableCell>{careerEventID}</TableCell>
+    <TableCell>{name}</TableCell>
+    <TableCell>{date}</TableCell>
+    <TableCell>{description}</TableCell>
+    <TableCell>{location}</TableCell>
+    <TableCell>{link}</TableCell>
+  </TableRow>
+);
+
+export default CareerEvent;
+
+//test to see how it looks
```

Comment:
What’s this? We’ll be testing the UI of the component in its test file if we still need this component.

Reply:
Deleted